code-challenges.js

```diff
@@ -7,42 +7,130 @@
 
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
+// 1. declare a variable
+// 2. set that variable to describe temperature
+// 3. set the variable to a data type that describes the temperature;
+//      in this case, set the variable to a number data type that describes what the temperature is at
+// 4. write a conditional statement to decide the output based on what the variable is set to;
+//      in this case, the output would be "below boiling", "at boiling", or "above boiling".
+// 5. since there are three possible outputs, write a if(){}else if () {}else {} conditional statement
+//      so if the temperature is 212 then the string outcome would be "X is at boiling,"
+//        if this statement is not true then the temperature is either below or above 212
+//        the if/elseif/else statement will continue to run until the condition is met
+//      if the temperature is below 212 then the string outcome would be "X is below boiling"
+//      but if the temperature is not 212 or below (aka greater than 212) then the string outcome would be "X is above boiling"
+// 6. test the conditional statement by declaring variables that are less than 212, equal to 212, and greater than 212
+//
+
+
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp = 35
 // var temp = 350
 // var temp = 212
 
-
-
-// --------------------2) Create the code that will combine two arrays and return the length using the variables provided below. Expected output: 10
+if (temp === 212) {
+  console.log("212 is at boiling point");
+// if the temperature is equal to 212 then print out the string "212 is at boiling point"
+}else if (temp === 35) {
+  console.log("35 is below boiling point");
+// if the temeprature is not equal to 212, continue running till conditional is met
+// if the temperature is 35 then print out the string "35 is below boiling point"
```

Comment:
You're on the right track here: I like your logic from earlier-- You don't just want 35 degrees to log is below the boiling point you want anything under 212 is below the boiling point. 
```javascript
else if (temp < 212) {
  console.log("your input is below boiling point");
```